BUILD.md

```diff
@@ -4,43 +4,84 @@
 
 To install cuML from source, ensure the following dependencies are met:
 
-1. [cuDF](https://github.com/rapidsai/cudf) (Same as cuML Version)
-2. zlib
-3. cmake (>= 3.26.4)
-4. CUDA (>= 12+)
-5. Cython (>= 0.29)
-6. gcc (>= 13.0)
-7. BLAS - Any BLAS compatible with cmake's [FindBLAS](https://cmake.org/cmake/help/v3.14/module/FindBLAS.html). Note that the blas has to be installed to the same folder system as cmake, for example if using conda installed cmake, the blas implementation should also be installed in the conda environment.
-8. clang-format (= 20.1.4) - enforces uniform C++ coding style; required to build cuML from source. The packages `clang=20` and `clang-tools=20` from the conda-forge channel should be sufficient, if you are on conda. If not using conda, install the right version using your OS package manager.
-9. NCCL (>=2.4)
-10. UCX [optional] (>= 1.7) - enables point-to-point messaging in the cuML standard communicator. This is necessary for many multi-node multi-GPU cuML algorithms to function.
+> **Note:** The easiest way to setup a fully functional build environment is to use the conda environment files located in `conda/environments/all_*.yaml`. These files contain all the dependencies listed below except for clang-format (only needed for development/contributing) and UCX (only needed for optional multi-node operations). To create a development environment, see the [recommended conda setup](#recommended-conda-setup) at the end of this section.
 
-It is recommended to use conda for environment/package management. If doing so, development environment .yaml files are located in `conda/environments/all_*.yaml`. These files contains most of the dependencies mentioned above (notable exceptions are `gcc` and `zlib`). To create a development environment named `cuml_dev`, you can use the follow commands:
+**Hardware Needed to Run cuML:**
+To run cuML code, you will need an NVIDIA GPU with compute capability 7.0 or higher (Volta™ architecture or newer). Note that while a GPU is not required to build or develop cuML itself, it is necessary to execute and test GPU-accelerated functionality.
```

Comment:
This is only true for CUDA 12. For CUDA 13 builds, minimum compute capability is 7.5. (check release notes [here](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html) to see removal of Volta support)

Reply:
I think we should is what I was suggesting.

Reply:
4a686a17c59cd324b0fee1cbcc175af69b10ecfc